Explain how to set errors when using manual instrumentation for .NET (#7186)
DataDog/documentation: content/en/tracing/manual_instrumentation/dotnet.md

}
```

### Setting Errors

> To recognize and mark errors that occur in your code, you can utilize the SetException method that is available to spans. In addition to marking the span as error, this will add [span metadata related to the error][4] to provide insight into the exception.

Reviewer: I think we should include the method signature and format it as `code`.
```suggestion
To recognize and mark errors that occur in your code, you can utilize the `Span.SetException(Exception)` method that is available to spans. In addition to marking the span as error, this will add [span metadata related to the error][4] to provide insight into the exception.
```

> To recognize and mark errors that occur in your code, you can utilize the SetException method that is available to spans. In addition to marking the span as error, this will add [span metadata related to the error][4] to provide insight into the exception.

Reviewer 2: ```suggestion
To recognize and mark errors that occur in your code, utilize the `SetException` method available to spans. The method marks the span as an error and adds [related span metadata][4] to provide insight into the exception.
```

Author: It won't let me commit this, so let me modify this to incorporate Lucas' wording around the method.

Author: @ruthnaebeck - can you check the latest version? I changed it to "To recognize and mark errors that occur in your code, utilize the `Span.SetException(Exception)` method available to spans. The method marks the span as an error and adds [related span metadata][4] to provide insight into the exception." , to make sure Lucas' changes are kept.
